src/components/view-cards.jsx

```diff
@@ -1,10 +1,48 @@
 import React from 'react';
 
 class ViewCards extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cards: null
+    }
+  }
+
+
+  componentDidMount() {
+    const cardsArray = this.props.cards;
+    const cards = cardsArray.map(card => {
+      return (
+        <div key={card.question} className="col-sm-4 mb-3">
+          <div className="card">
+            <div className="card-header">
+              Question:
+              <div>
+                {card.question}
+              </div>
+            </div>
+            <div className="card-body">
+              Answer:
+              <div>
+                {card.answer}
+              </div>
+            </div>
+          </div>
+        </div>
+      );
+    });
+    this.setState( {cards} );
+  }
```

Comment:
Instead of storing these in state, just return them and call your method in `render()`